### PR TITLE
Disease and Protein Browser: Code Refactoring

### PR DESCRIPTION
**Pull request by suhana13**
In this PR, I have added a biomedical/ subdirectory for the css files and the protein and disease folders under the biomedical subdirectory. Please let me know if more changes are needed!

### CODE REVIEW

**Comment by beets** [comment]
Can you please give the PR a more descriptive title?

File: static/js/biomedical/bio_charts_utils.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+export const DEFAULT_BRIGHTEN_PERCENTAGE = "112%";
+// x axis label shift
+export const X_LABEL_SHIFT = 40;
+export type Datum =
+  | DiseaseGeneAssociationData
+  | ProteinNumData
+  | ProteinNode
+  | InteractionLink
+  | DiseaseAssociationType;
+/**
```
Comment (chejennifer) [inline static/js/biomedical/bio_charts_utils.ts:43]: add an empty line before this function

File: static/js/biomedical/bio_charts_utils.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+): void {
+  // brighten element: https://stackoverflow.com/a/69610045
+  d3.select(`#${elementID}`).style(
+    "filter",
+    `brightness(${brightenPercentage})`
+  );
+  // update tooltip text
+  TOOL_TIP.html(toolTipText);
+  // show tooltip
+  TOOL_TIP.style("opacity", 1);
```
Comment (chejennifer) [inline static/js/biomedical/bio_charts_utils.ts:59]: I believe you can chain this with the previous line like:

```
TOOL_TIP.html(toolTipText).style("opacity", 1);

File: static/js/biomedical/bio_charts_utils.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+}
+/**
```
Comment (chejennifer) [inline static/js/biomedical/bio_charts_utils.ts:61]: please add an empty line between functions

File: static/js/biomedical/bio_charts_utils.ts (hunk 1 above)
Comment (suhana13) [reply static/js/biomedical/bio_charts_utils.ts:44]: Done!

File: static/js/biomedical/bio_charts_utils.ts (hunk 2 above)
Comment (suhana13) [reply static/js/biomedical/bio_charts_utils.ts]: Yeap, linked the two lines. Thanks for the suggestion!

File: static/js/biomedical/bio_charts_utils.ts (hunk 3 above)
Comment (suhana13) [reply static/js/biomedical/bio_charts_utils.ts:61]: Done! Although, I thought the running the linter should also fix issues like adding extra lines, perhaps not in this case:(

**Review by chejennifer — COMMENTED** [review]
Thanks for the updates, just one more commenting fix!

**Review by beets — COMMENTED** [review]
this is nice - thanks for the refactor!

**Comment by rogershijin** [comment]
Wow amazing refactor @suhana13 ! Biomedical DC directory looks beautiful, and hats off to you for figuring out CSS imports!

**Review by chejennifer — APPROVED** [review]
thanks for all the changes!

**Pull request merged**